src/containers/BooksForm.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const INPUT = styled.input`
+  padding: 7px;
+  margin: 2% 3%;
+  margin-left: -15%;
+`;
+
+const SELECT = styled.select`
+  padding: 5px;
+  background: green;
+  color:white;
+  margin-right: 2%;
+`;
```

Comment:
[Optional] We can move all these styled-components to another file and give a cleaner look.